tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-var-type.bal

```diff
@@ -530,6 +530,42 @@ function testUsingAnIntersectionTypeInQueryExpr2() {
     assertEquality([{email : "peter@abc.com", teamId: 2}], j);
 }
 
+function testBlangMethodCallsOfQueryReturn() {
```

Comment:
Why are we saying BlangMethodCalls? We can just say something like `testMethodCallExprsOnQueryExprs` IMO.

Reply:
Updated